src/serviceWorker/index.ts

```diff
@@ -21,14 +21,21 @@ registerRoute(new DocumentRoute())
 const { onDemandEntries, precacheEntries } = groupEntries(self.__WB_MANIFEST)
 const onDemandURLs = onDemandEntries.map((entry) => (typeof entry === 'string' ? entry : entry.url))
 
+const onDemandCacheName = `${cacheNames.prefix}-on-demand-${cacheNames.suffix}`
 registerRoute(
   new Route(
     ({ url }) => onDemandURLs.includes('.' + url.pathname),
     new CacheFirst({
-      cacheName: 'media',
-      plugins: [new ExpirationPlugin({ maxEntries: 16 })],
+      cacheName: onDemandCacheName,
+      plugins: [new ExpirationPlugin({ maxEntries: 64 })],
```

Comment:
how did you pick this number?

Reply:
I navigated to all tabs on the navbar, and then checked how many resources had been cached: 33.
I then rounded up to the nearest power of 2: 64.